src/utils/constants.js

```diff
@@ -266,6 +267,15 @@ export const DEFAULT_OPTIONS_PATH = {
 
 export const DEFAULT_OPTIONS_POINTCLOUD = {};
 
+
+export const DEFAULT_OPTIONS_POINT = {
+    color: '#ff0000',
+    alpha: 1,
+    radius: 0.2,
+    widthSegments: 32,
```

Comment:
As mentioned above the primitive segments are currently not configurable by the user.
`widthSegments` and `heightSegments` can be removed.
The `Point` class shouldn't be using them. Rather the `Sphere` primitive can directly use the default value for now.